### PR TITLE
First Contribution: I added a circle to the confetti, I think

### PR DESCRIPTION
**Pull request by JHMuir**

### CODE REVIEW

File: static/assets/1st/circle-thing.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<circle cx="25" cy="75" r="20"/>
```
Comment (EthanThatOneKid) [inline static/assets/1st/circle-thing.svg]: > [Hi did I do it right?](https://github.com/EthanThatOneKid/acmcsuf.com/pull/672/commits/359c23a02aa36ea30a0baf54e815cd063d25e1d4)

You were close!

SVG files must begin with a root tag of `svg`.

```suggestion
<svg width="50" height="50" viewBox="0 0 50 50" fill="#2c91c6" xmlns="http://www.w3.org/2000/svg">
  <circle cx="25" cy="25" r="25"/>
</svg>
```

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
There is only a minor change keeping you from saying, "first try 😎", but luckily it is a super minor change needed to make your SVG file valid. I left a suggestion in my review, feel free to commit the suggestion directly or add it manually if you have additional changes :).

**Comment by jaasonw** [comment]
Also for future reference, your pull requests should contain the following
image

**Review by EthanThatOneKid — APPROVED** [review]
Nice first contribution!

**Pull request merged**